packages/frontend-2/components/model/Card.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <!-- eslint-disable-next-line vuejs-accessibility/mouse-events-have-key-events -->
+  <div
+    class="group bg-background hover:bg-primary rounded-xl shadow hover:shadow-xl hover:scale-[1.04] transition-all dark:shadow-slate-500/10 hover:cursor-pointer"
+    @mouseenter="hover = true"
+    @mouseleave="hover = false"
```

Comment:
I haven't seen visually what the effect you're going for is, but maybe the 'hover:' classes are enough?

Reply:
cleanup needed, these were experiments